### PR TITLE
docs: add Render hosting guide

### PR DESCRIPTION
**Pull request by anurag**
## Motivation

[Render](https://render.com) is an official deployment option for [create-react-app](https://facebook.github.io/create-react-app/docs/deployment#render-https-rendercom), [gatsby](https://www.gatsbyjs.org/docs/deploying-to-render/), and [hugo](https://gohugo.io/hosting-and-deployment/hosting-on-render/). This PR adds a guide for Docusaurus users who may want to host their site on Render.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/Docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes.

## Test Plan
- [x] `yarn prettier && yarn lint`
- [x] `yarn test`
- [x] Visual inspection of all doc versions.

### CODE REVIEW

**Review by yangshun — CHANGES_REQUESTED** [review]
Sorry for the late review. The PR looks ok but can we leave out updating the old versioned docs? The whole intention of versioning is to freeze old versions and new stuff should only go into new versions.

**Comment by anurag** [comment]
Makes sense and will submit an update shortly.

**Comment by anurag** [comment]
> Sorry for the late review. The PR looks ok but can we leave out updating the old versioned docs? The whole intention of versioning is to freeze old versions and new stuff should only go into new versions.

@yangshun PR updated.

File: website/docs/deployment.md [hunk 1]
```diff
@@ -76,6 +76,33 @@ References:
...
+
+2. Select the branch to deploy. The default is `master`.
+
+2. Enter the following values during creation.
+
+    |  Field  |  Value |
+    | ------- | ----- |
+    | **Environment** | `Static Site` |
+    | **Build Command** | `cd website; yarn install; yarn build` |
+    | **Publish Directory** | `website/build/<projectName>` |
```
Comment (endiliey) [inline website/docs/deployment.md]: this is no longer true for docusaurus 2. projectname is not needed anymore

**Review by endiliey — CHANGES_REQUESTED** [review]
deploying for docusaurus 2 is slightly different.

**Review by endiliey — APPROVED** [review]
lgtm

**Review by yangshun — APPROVED** [review]
Thank you!

**Comment by slorber** [comment]
FYI, we are going to clean up our existing Docusaurus docs deployment page in favor of a GitHub discussions category. 

This means your platform won't appear anymore on this page: https://docusaurus.io/docs/deployment

If you want your hosting / CI / CDN platform to remain documented for Docusaurus users, please submit an entry at: https://github.com/facebook/docusaurus/discussions/categories/deployment-platforms

This will be your responsibility to keep it up to date over time.

**Pull request merged**